core/src/main/java/org/apache/metamodel/query/FromItem.java

```diff
@@ -222,7 +222,7 @@ public String toStringNoAlias(boolean includeSchemaInColumnPaths) {
         }
         StringBuilder sb = new StringBuilder();
         if (_table != null) {
-            if (_table.getSchema() != null && _table.getSchema().getName() != null) {
+            if (_table.getSchema() != null && _table.getSchema().getName() != null && !_table.getSchema().getName().isEmpty()) {
```

Comment:
I'm not sure what the code style conventions are for MetaModel, but this line is really long. Maybe change it to:

```
            if (_table.getSchema() != null && _table.getSchema().getName() != null 
                    && !_table.getSchema().getName().isEmpty()) {
```